charts/monochart/values.example.yaml

```diff
@@ -131,7 +125,16 @@ deployment:
           - key: Name
             operator: In
             values:
-            - va-verity-eks--stage-vng-public
+            - custom-worker-group
+
+persistence:
```

Comment:
Rely on the existing persitence object to define whether we want a disk and if this disk should be ephemeral or not